### ISSUE TITLE
Test results inconsistent

### ISSUE THREAD

**Issue by zhanganny** [saidwivedi/TokenHMR#12]
Hello, I try to evaluate the released original model (used in the paper). But I get different results as below, which is not as good as the results showed in the paper. I tested with PyTorch 2.1.0 and CUDA 11.8 and use the same environment as readme.md descirbed.

<img width="1498" alt="image" src="https://github.com/user-attachments/assets/100835b7-e872-49c5-b54b-2ac31c3ad016">
<img width="729" alt="image" src="https://github.com/user-attachments/assets/fde798e6-4282-4627-8bf1-5513637b2cbe">

**Comment by saidwivedi** [comment]
I ran the evals on my local environment and it matches with the one reported in the original submission (see below). 
![3DPW Evals](https://github.com/user-attachments/assets/9d94b679-8c4f-4617-9f81-f1233050868a)

The only difference is that public release version of TokenHMR has different environment (newer PyTorch and CUDA) than I used during the submission. The difference is minor between submitted version `(PVE: 84.6, MPJPE: 71.0, PA-MPJPE 44.3)` and release version `(PVE: 84.8, MPJPE: 72.0, PA-MPJPE 45.5)` for 3DPW dataset.

Nevertheless, I will update the arXiv to reflect the numbers in the release version. Thank you for pointing it out.
